h2/src/tools/org/h2/build/Build.java

```diff
@@ -404,9 +406,9 @@ private void downloadOrVerify(boolean offline) {
         downloadOrVerify("ext/org.osgi.core-" + OSGI_VERSION + ".jar",
                 "org/osgi", "org.osgi.core", OSGI_VERSION,
                 "6e5e8cd3c9059c08e1085540442a490b59a7783c", offline);
-        downloadOrVerify("ext/org.osgi.enterprise-" + OSGI_VERSION + ".jar",
-                "org/osgi", "org.osgi.enterprise", OSGI_VERSION,
-                "4f6e081c38b951204e2b6a60d33ab0a90bfa1ad3", offline);
+        downloadOrVerify("ext/org.osgi.service.jdbc-" + OSGI_JDBC_VERSION + ".jar",
+                "org/osgi", "org.osgi.service.jdbc", OSGI_JDBC_VERSION,
+                "074e3f65d4496905b8ad5db092413223e16ac7f2", offline);
```

Comment:
Build fails due to incorrect checksum, it should be 07673601d60c98d876b82530ff4363ed9e428c1e for this jar from Maven Central.